packages/editor/src/services/enum.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-unused-vars */
+enum ExploreMenuTabs {
```

Comment:
typo: explore -> explorer

Reply:
BTW, I think it's an eslint bug which shows enum as unused variables.

Reply:
> typo: explore -> explorer

corrected

Reply:
> BTW, I think it's an eslint bug which shows enum as unused variables.

yes, can use `"@typescript-eslint/no-unused-vars": ["error"]` instead of `no-unused-vars` to solve this problem